images/timer.proto

```diff
@@ -21,6 +21,7 @@ message posix_timer_entry {
 	required uint64		insec		= 8;
 	required uint64		vsec		= 9;
 	required uint64		vnsec		= 10;
+	required int32		notify_thread_id= 11;
```

Comment:
this has to be "optional". Images have to be backward-compatible.

Reply:
I think users should dump and restore using the same version of criu.
If we don't save notify_thread_id in images, recovery won't success at all.

Reply:
> I think users should dump and restore using the same version of criu.

We declare that criu restore should work for images of the same or lower version.

> If we don't save notify_thread_id in images, recovery won't success at all.

It won't succeed only in the case of SIGEV_THREAD_ID, will it? Actually, notify_thread_id means something only in the SIGEV_THREAD_ID case, and this looks like another reason why it has to be optional.